serverlessworkflow/modules/ROOT/pages/cloud/index.adoc

```diff
@@ -98,6 +98,16 @@ xref:cloud/operator/build-and-deploy-workflows.adoc[]
 Learn how to build and deploy workflow services with {operator_name}
 --
 
+[.card]
+--
+[.card-title]
+xref:cloud/operator/workflow-dev-profile-builder-configurable.adoc[]
```

Comment:
can't we have this as part of the dev profile section instead add a new one?

Reply:
I agree. It's better to add a section to the Dev profile section.

Reply:
Done